src/AnimatedLineGraph.tsx

```diff
@@ -215,16 +229,22 @@ export function AnimatedLineGraph({
     [isActive, setIsActive]
   )
   const positions = useDerivedValue(
-    () => [
-      0,
-      Math.min(0.15, pathEnd.current),
-      pathEnd.current,
-      pathEnd.current,
-      1,
-    ],
+    () => [0, pathEnd.current, pathEnd.current, pathEnd.current, 1],
     [pathEnd]
   )
 
+  useEffect(() => {
+    if (
+      typeof initialIndex === 'number' &&
+      initialIndex < points.length &&
+      width
+    ) {
+      const xForIndex = Math.round((initialIndex * width) / points.length)
+      setFingerX(xForIndex)
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Why?

Reply:
The linter was complaining about the `setFingerX` function, but we dont want it to be listed here.

Reply:
> but we dont want it to be listed here

Why not?

Reply:
Supposing we are passing an anonymous function in the `onPointSelected` prop, dont we risk running in some sort of infinite loop ?